detox/src/artifacts/CloudArtifactsManager.js

```diff
@@ -0,0 +1,76 @@
+class CloudArtifactsManager {
```

Comment:
We have created this ArtifactsManager class because artifacts are being managed by us and we don't want any logs related to artifacts management in our tester logs. 
All the methods here are the callbacks for the events happening throughout the test cycle